react/components/BaseProductQuantity.tsx

```diff
@@ -57,7 +60,12 @@ const BaseProductQuantity: StorefrontFunctionComponent<BaseProps> = ({
   }
 
   const showAvailable = availableQuantity <= warningQuantityThreshold
-
+  let unitMultiplier
+  if (quantitySelectorStep === 'singleUnit') {
+    unitMultiplier = 1
+  } else {
+    unitMultiplier = selectedItem.unitMultiplier
+  }
```

Comment:
This could be simplified into something like:

```suggestion
  const unitMultiplier = quantitySelectorStep === 'singleUnit' ? 1 : selectedItem.unitMultiplier
  
```

Reply:
The lint will probably complain that this line is too long, so I would recommend typing it out in your editor and letting ESLint/Prettier do the magic